source/content/new-relic.md

```diff
@@ -27,16 +21,59 @@ Select the **New Relic** tab on your Site Dashboard, and click **Activate New Re
 
 New Relic is automatically enabled for all application containers added to the site, including Multidev environments.
 
+## New Relic Access
+Go to the New Relic workspace on your Dashboard, and click **Go to New Relic**.
+
+### Who is the New Relic account associated with?
+
+* If the site owner is a **user**, the site owner's name and email address is used to create the New Relic account.
+* If the site owner is an **organization**, the name and email address of the organization's Pantheon user who activates New Relic is used to create the New Relic account.
+* Your New Relic **password** is unique to your New Relic account. If you have forgotten or don't yet have a New Relic password, use your Pantheon account email to request a [password reset](https://login.newrelic.com/passwords/forgot).
```

Comment:
@carolyn-shannon this line is new to the doc with this introduction. Not sure if it was in preparation for NR changes, or currently true.

Reply:
Good catch. This was based on an earlier conversation but no longer seems to be the case. Can be removed.